staging/src/k8s.io/kubectl/pkg/cmd/expose/expose_test.go

```diff
@@ -335,7 +335,7 @@ func TestRunExposeService(t *testing.T) {
 			},
 			flags: map[string]string{"selector": "svc=frompod", "port": "90", "labels": "svc=frompod", "generator": "service/v2"},
 			output: &corev1.Service{
-				ObjectMeta: metav1.ObjectMeta{Name: "a-name-that-is-toooo-big-for-a-service-because-it-can-only-handle-63-characters", Namespace: "", Labels: map[string]string{"svc": "frompod"}},
+				ObjectMeta: metav1.ObjectMeta{Name: "a-name-that-is-toooo-big-for-a-service-because-it-can-only-handle-63-characters"[:63], Namespace: "", Labels: map[string]string{"svc": "frompod"}},
```

Comment:
Hmm?

Reply:
This test is actually broken. `test.output` in the table tests was supposed to be used somewhere, but it was never plumbed through. It is hard to tell how these table tests worked previously without this..

But anyway, after I fixed the plumbing for `test.output`, I had to fix this specific test, which is testing to check that the `.metadata.name`.

Reply:
Okay looking back, I see that what I did was changed this to use the resource.Helper instead of our DynamicClient. Very interesting but that change somehow revealed that these tests are broken...

I could change back to use the DynamicClient maybe..